joomla/joomla-cms
# [5.0] Update JSON flags for rendering slashes and unicode, and debugging

### Summary of Changes

Set JSON flags for rendering slashes and unicode, and debugging.
For script options and schema.org


### Testing Instructions
Apply patch,
Check source code of any page.
Script options should be rendered without escaped slashes.



### Link to documentations
Please select:
